.github/workflows/llm_bench-python.yml

```diff
@@ -98,13 +99,6 @@ jobs:
         optimum-cli export openvino --trust-remote-code --model openai/whisper-tiny ./ov_models/whisper-tiny
         python ./tools/llm_bench/benchmark.py -m ./ov_models/whisper-tiny --media multilingual_librispeech/data/mls_polish/train/audio/3283_1447_000/3283_1447_000000.flac -d cpu -n 1
         python ./tools/llm_bench/benchmark.py -m ./ov_models/whisper-tiny --media multilingual_librispeech/data/mls_polish/train/audio/3283_1447_000/3283_1447_000000.flac -d cpu -n 1 --genai
-    - name: WWB Tests
```

Comment:
why tests are removed? just to ignore red status?

Reply:
It was a duplicate. Stateful job runs the same